src/bcrhp/bcrhp/pkg/map_layers/mapbox_spec_json/overlays/nts50k/nts50k.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This layer is to be removed so we probably don't need to copy it over.

Reply:
Agreed that we'll leave it here for now until next release. Need a PR to remove the unused map layers in the next release.